<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Texas Hold'em Poker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <!-- Graphics view to display cards -->
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>580</width>
      <height>380</height>
     </rect>
    </property>
   </widget>
   <!-- New Game Button -->
   <widget class="QPushButton" name="newGameButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>New Game</string>
    </property>
   </widget>
   <!-- Deal Flop Button -->
   <widget class="QPushButton" name="dealFlopButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>400</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Deal Flop</string>
    </property>
   </widget>
   <!-- Deal Turn Button -->
   <widget class="QPushButton" name="dealTurnButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>400</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Deal Turn</string>
    </property>
   </widget>
   <!-- Deal River Button -->
   <widget class="QPushButton" name="dealRiverButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>400</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Deal River</string>
    </property>
   </widget>
   <!-- Determine Winner Button -->
   <widget class="QPushButton" name="determineWinnerButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>400</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Determine Winner</string>
    </property>
   </widget>
   <!-- Exit Button -->
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>440</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
